src/Gui/CommandFeat.cpp

```diff
@@ -144,22 +144,27 @@ StdCmdToggleFreeze::StdCmdToggleFreeze()
 void StdCmdToggleFreeze::activated(int iMsg)

```

Comment:
line 134 above has a typo:
+ QT_TR_NOOP("Toggles freeze **sate** of the selected objects. A freezed object is not recomputed when its parents change.")

should be **state**